### ISSUE TITLE
PHP 5.6 compatibility

### ISSUE THREAD

**Issue by oparoz** [owncloud/gallery#20]
The app may not be compatible with PHP 5.6.

In order to test it on PHP 5.6, you need to edit `apps/galleryplus/appinfo/info.xml`
Change

```
<php min-version="5.4" max-version="5.x"/>
```

to

```
<php min-version="5.4" max-version="5.7"/>
```

**Comment by leandro-lucarella-sociomantic** [comment]
With the current max-version="5.5" I also owncloud complains about needing version **lower** than 5.5, when using PHP 5.5.

**Comment by leandro-lucarella-sociomantic** [comment]
Manually changing the `info.xml` file to use `max-version="5.6"` makes it work for PHP 5.5.

**Comment by oparoz** [comment]
It's been reported as a bug in `core` [here](https://github.com/owncloud/core/issues/14392#issuecomment-76146940), but I'll update master asap as I'm guessing this won't be fixed before 8.1.
